xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-default/src/main/java/org/xwiki/refactoring/internal/DefaultModelBridge.java

```diff
@@ -213,7 +216,7 @@ public void createRedirect(DocumentReference oldReference, DocumentReference new
             try {
                 XWikiDocument oldDocument = xcontext.getWiki().getDocument(oldReference, xcontext);
                 int number = oldDocument.createXObject(redirectClassReference, xcontext);
-                String location = this.compactEntityReferenceSerializer.serialize(newReference, oldReference);
+                String location = this.localEntityReferenceSerializer.serialize(newReference);
```

Comment:
I'm not sure about the *local* entity reference serializer. What if I move a page to a different wiki?